countries/views/answer.erb

```diff
@@ -0,0 +1,5 @@
+
+
+<img src="<%= @question.flag %>">
```

Comment:
It would be good to put a size constraint here so that the images are a uniform size -- you use the `height` attribute elsewhere, or you could put a class on this element and add some css.